### DISCUSSION TITLE
[Question]: How do I switch to the new web interface code?

### DISCUSSION THREAD

**Discussion by apbarratt** [blakeblackshear/frigate#10051] [Ask A Question]
### What is your question:

I figured out how to fix the flex box on the config screen to prevent the need to scroll the main window to see the whole editor.  So I forked Frigate with the intention of submitting a new PR but found there are actually two `web` folders and the code I'm seeing rendered by my Add-On in Home Assistant is from the one called `/web-old`.

I'm assuming there's some way to switch my setup to try out the new `/web` folder's code.  But I'm not seeing any obvious mention of it in the documentation.  Could I be pointed in the right direction?

Also, could you tell me what the time line for retiring `/web-old` is?  As if it's a long term thing I might as well still create that flex-box PR for the old config page :)

**Comment by NickM-27** [comment]
The entire webui is being rewritten for 0.14. It's not ready for general users. We've already fixed the issue with the config page in the new webui
